### PR TITLE
fix bline point order when saving for older canvas (< synfig 1.4)

### PR DESCRIPTION
**Pull request by rodolforg**
this is due to changes discussed here https://github.com/synfig/synfig/issues/1284

They will still have trouble in 1.3.x versions :/

### CODE REVIEW

File: synfig-core/src/synfig/savecanvas.cpp [hunk 1]
```diff
@@ -579,10 +579,16 @@ xmlpp::Element* encode_dynamic_list(xmlpp::Element* root,ValueNode_DynamicList::
...
 	ValueNode_DIList::ConstHandle dilist_value_node(ValueNode_DIList::ConstHandle::cast_dynamic(value_node));
 
+	bool must_rotate_point_list = false;
+
 	if(bline_value_node)
 	{
 		if(bline_value_node->get_loop())
+		{
 			root->set_attribute("loop","true");
+			if (save_canvas_version < RELEASE_VERSION_1_4_0) // or get_file_version()?
```
Comment (rodolforg) [inline synfig-core/src/synfig/savecanvas.cpp:589]: @blackwarthog could you point me the difference between `save_canvas_version` and `get_file_version()` ?
Can they store different values or are they always the same?

**Comment by morevnaproject** [comment]
Great addition! 

I think it is okay if users still have trouble with 1.3.x, since this is a development version (i.e. "unoffilcial/unstable" release). ^__^

**Comment by morevnaproject** [comment]
It works perfectly, thank you so much! ^__^

File: synfig-core/src/synfig/savecanvas.cpp (hunk 1 above)
Comment (morevnaproject) [reply synfig-core/src/synfig/savecanvas.cpp:589]: `get_file_version()` returns `save_canvas_version`, so I think they are always the same.

**Comment by morevnaproject** [comment]
Oh, I just remembered that we also need to lower canvas version when saving file in old version.

Otherwise following situation is possible:
- user opens (creates) file in Synfig 1.4
- adds looped advanced outlines with widthpoints
- user saves file in older format
- user opens closes file and opens it again in Synfig 1.4 -> widthpoint positions are wrong.

I have added PR #1365 about that. ^__^

**Comment by rodolforg** [comment]
Nice catch :)

**Pull request merged**